website/content/docs/discovery/dns.mdx

```diff
@@ -353,25 +358,49 @@ $ echo -n "20010db800010002cafe000000001337" | perl -ne 'printf join(":", unpack
 
 By default, all service lookups use the `default` namespace
 within the partition and datacenter of the Consul agent that received the DNS query.
-To lookup services in another namespace, partition, and/or datacenter,
+To lookup services in another namespace, partition, datacenter, or cluster peer,
```

Comment:
Intended to support in the future, but no support now since it's always parsed as dc